sdk/security/keyvault/azkeys/ci.yml

```diff
@@ -28,13 +28,11 @@ extends:
     ServiceDirectory: 'security/keyvault/azkeys'
     RunLiveTests: true
     UsePipelineProxy: false
-    AdditionalMatrixConfigs:
-      - Name: keyvault_test_matrix_addons
-        Path: sdk/security/keyvault/azkeys/platform-matrix.json
-        Selection: sparse
-        GenerateVMJobs: true
 
     # Due to the high cost of Managed HSMs, we only want to test using them weekly.
-    ${{ if not(contains(variables['Build.DefinitionName'], 'tests-weekly')) }}:
-      MatrixFilters:
-        - ArmTemplateParameters=^(?!.*enableHsm.*true)
+    ${{ contains(variables['Build.DefinitionName'], 'weekly') }}:
```

Comment:
```suggestion
    ${{ if contains(variables['Build.DefinitionName'], 'weekly') }}:
```

Reply:
fixed in [f08cdd1](https://github.com/Azure/azure-sdk-for-go/pull/22855/commits/f08cdd1785163ec5420326646c86570d13f69ff8)